addon/components/tf-fields/label.js

```diff
@@ -0,0 +1,29 @@
+/**
+  @module ember-ticketfly-fields
```

Comment:
Even though we _can_ have a nested component without a dash, it may make compatibility with [Module Unification](https://github.com/dgeb/rfcs/blob/module-unification/text/0000-module-unification.md) weird because how that nested component is invoked can change...so currently you have:

```hbs
{{tf-fields/label label hideLabel=hideLabel for=inputId}}
```

But, in the future, it would just be:

```hbs
{{label label hideLabel=hideLabel for=inputId}}
```

...because it is aware of its parent namespace. But, can't have a component without a dash.